composer.json

```diff
@@ -11,28 +11,25 @@
     }
   ],
   "require": {
-    "php": "^7.2",
-    "illuminate/contracts": "5.6.* || 5.7.* || 5.8.* || ^6 || ^7 || ^8",
-    "illuminate/http": "5.6.* || 5.7.* || 5.8.* || ^6 || ^7 || ^8",
-    "thecodingmachine/safe": "^1.1.3",
-    "webonyx/graphql-php": "^0.13.2 || ^14"
+    "php": "^7.2 || ^8.0",
+    "illuminate/contracts": "5.6.* || 5.7.* || 5.8.* || ^6.0 || ^7.0 || ^8.0",
+    "illuminate/http": "5.6.* || 5.7.* || 5.8.* || ^6.0 || ^7.0 || ^8.0",
+    "thecodingmachine/safe": "^1.1",
+    "webonyx/graphql-php": "^0.13.2 || ^14.4"
   },
   "require-dev": {
-    "ergebnis/composer-normalize": "^2.11.0",
-    "infection/infection": "~0.20.2",
-    "jangregor/phpstan-prophecy": "0.8.1",
-    "orchestra/testbench": "3.6.* || 3.7.* || 3.8.* || 3.9.* || ^4 || ^5 || ^6",
-    "phpstan/extension-installer": "^1",
-    "phpstan/phpstan": "0.12.57",
-    "phpstan/phpstan-deprecation-rules": "0.12.5",
-    "phpstan/phpstan-strict-rules": "0.12.5",
-    "phpunit/phpunit": "^7.5 || ^8.4",
-    "thecodingmachine/phpstan-safe-rule": "v1.0.1"
+    "ergebnis/composer-normalize": "^2.11",
+    "infection/infection": "~0.15",
+    "jangregor/phpstan-prophecy": "^0.8.1",
```

Comment:
Prefer to keep those locked at a specific version.

Reply:
Lock for specific version will not able to receive any updates for that package and this will also increase security risk. Unless there is compatible issue, I don't think it is a good practice.